datastream-server/src/main/java/com/linkedin/datastream/server/DatastreamTaskImpl.java

```diff
@@ -310,14 +310,16 @@ public void acquire(Duration timeout) {
       // Need to confirm the dependencies for task are not locked
       _dependencies.forEach(predecessor -> {
         if (_zkAdapter.checkIsTaskLocked(this.getConnectorType(), this.getTaskPrefix(), predecessor)) {
-          String msg = String.format("previous task %s failed to release lock in %dms", predecessor, timeout.toMillis());
+          String msg = String.format("previous task %s failed to release lock in %dms for task %s", predecessor,
```

Comment:
is predecessor meant to be logged in full

Reply:
the `_dependencies` list is a list of Strings, and the predecessor is just  the task name. So we're good here